src/AbpHelper.Core/Templates/Crud/Groups/Service/src/{{ProjectInfo.FullName}}.Application.Contracts/{{EntityInfo.RelativeDirectory}}/I{{EntityInfo.Name}}AppService.cs

```diff
@@ -13,6 +14,6 @@ public interface I{{ EntityInfo.Name }}AppService :
             {{ DtoInfo.CreateTypeName }},
             {{ DtoInfo.UpdateTypeName }}>
     {
-
+        Task<PagedResultDto<{{ EntityInfo.Name }}Dto>> GetListByFilterAsync({{ DtoInfo.GetTypeName }} input);
```

Comment:
Why not change `PagedAndSortedResultRequestDto` to `{ DtoInfo.GetTypeName }}`? I think we don't need an extra method definition.

Reply:
yes, I aggree with you